packages/fluentui/docs/src/components/ComponentBestPractices.tsx

```diff
@@ -22,7 +22,7 @@ const ComponentBestPractices: React.FC<ComponentBestPracticesProps> = ({ doList,
   const updatedDontList: ShorthandCollection<ListItemProps> = [];
 
   if (_.size(doList) > 0) {
-    _.map(doList, element => {
+    _.map(doList, (element: string) => {
```

Comment:
Is there a problem using `ShorthandValue` ? 

```suggestion
    _.map(doList, (element: ShorthandValue<ListItemProps>) => {
```